tests/integration/nmstatectl_test.py

```diff
@@ -97,6 +113,17 @@
 ]
 
 
+@pytest.fixture(scope="module", autouse=True)
+def enable_lo_ipv6():
+    with open("/proc/sys/net/ipv6/conf/lo/disable_ipv6") as fd:
+        old_state = fd.read().strip()
+    with open("/proc/sys/net/ipv6/conf/lo/disable_ipv6", "w") as fd:
+        fd.write("0")
```

Comment:
I think you can do it just opening the file one time.

Reply:
Yes. Changed to use `r+` mode.